common/vehicle_constants_manager/package.xml

```diff
@@ -10,14 +10,14 @@
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
   <buildtool_depend>autoware_auto_cmake</buildtool_depend>
 
+  <depend>autoware_auto_common</depend>
   <depend>rclcpp</depend>
   <depend>rclcpp_components</depend>
-  <depend>autoware_auto_common</depend>
 
   <test_depend>ament_cmake_gtest</test_depend>
-  <!-- <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend> -->
   <test_depend>autoware_auto_cmake</test_depend>
+  <!-- <test_depend>ament_lint_auto</test_depend>
```

Comment:
cc @KeisukeShima We have to add a test case. I'll create an issue in https://github.com/tier4/pre-commit-hooks-ros.